packages/now-next/test/fixtures/00-i18n-support-root-catchall/additional.js

```diff
@@ -121,14 +187,27 @@ module.exports = function (ctx) {
     expect($('#router-locale').text()).toBe('nl-NL');
 
     // wait for revalidation to occur
-    await new Promise(resolve => setTimeout(resolve, 4000));
+    await waitFor(2000);
 
     const res2 = await fetch(`${ctx.deploymentUrl}/nl-NL/second`);
     expect(res2.status).toBe(200);
 
     $ = cheerio.load(await res2.text());
-    const props2 = JSON.parse($('#props').text());
-    expect(initialRandom).not.toBe(props2.random);
     expect($('#router-locale').text()).toBe('nl-NL');
+
+    await check(async () => {
+      const res = await fetch(`${ctx.deploymentUrl}/nl-NL/second`);
+
+      if (res.status !== 200) {
+        throw new Error(`Invalid status code ${res.status}`);
+      }
+      const $ = cheerio.load(await res.text());
+      const props = JSON.parse($('#props').text());
+
+      if (isNaN(props.random)) {
+        throw new Error(`Invalid random value ${props.random}`);
+      }
+      return initialRandom !== props.random ? 'success' : 'failed';
+    }, 'success');
```

Comment:
Can this be a shared function like `checkRandomChanged()` to avoid copy/pasta?

Reply:
Added a function to de-dupe this a bit